client/src/reader/components/control-menu/AppearanceMenuBody.js

```diff
@@ -247,29 +267,48 @@ class AppearanceMenuBody extends Component {
       });
 
     return (
-      <fieldset className="appearance-menu__radio-group">
-        <legend className="control-menu__legend">
+      <div className="appearance-menu__radio-group" role="group">
+        <div className="control-menu__legend">
           {this.props.t("reader.menus.appearance.color_scheme")}
-        </legend>
-        {this.colorSchemeOptions.map(option => (
-          <label key={option.value} className={labelClassName(option)}>
-            <input
-              value={option.value}
-              name="color-scheme-radios"
-              type="radio"
-              checked={option.value === this.colorScheme}
-              onChange={this.handleColorSchemeControl}
-              className="appearance-menu__radio-input"
-            />
-            <Utility.IconComposer
-              icon="CheckUnique"
-              size={30}
-              className="appearance-menu__color-scheme-icon"
-            />
-            <span className="screen-reader-text">{option.label}</span>
-          </label>
-        ))}
-      </fieldset>
+        </div>
```

Comment:
If you switch this from a fieldset/legend to divs, then you need to restore the association of the label with the group using `aria-labelledby`.